client/scss/components/_button.scss

```diff
@@ -355,4 +355,17 @@
       }
     }
   }
+
+  // Ensure visual consistency between button and a elements in WHCM
+
+  @media (forced-colors: active) {
+    &:not(:disabled):not(.disabled):not([disabled]) {
```

Comment:
```suggestion
    &:not(:disabled),
    &:not(.disabled),
    &:not([disabled]) {
```

Just a minor tweak here, this will make three separate selectors instead of one very specific one. 

I have not run locally though and will try to test this out in the next few days. Otherwise another core team member will have a look. 

Thanks so much for putting this PR up.

Reply:
Unfortunately, this particular solution doesn't fix the bug entirely and enforce the needed result from all types of buttons: several disabled buttons get `ButtonText` instead of `GreyText`, and some Icons get `LinkText`. Hovering gets messy accordingly.

I'm honestly still in the process of studying how exactly all these selectors work together and had my fair share of trying and failing. The solution in PR works though.

<img width="539" alt="image" src="https://user-images.githubusercontent.com/51043550/195879824-c838d126-3eff-405b-ad1a-962fe5e1d00c.png">

Reply:
Good call - yeah the CSS probably does need to be this specific here unfortunately. 

I'll try to take a look locally.